packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -455,6 +455,10 @@ export class SDKClient {
             if (sdkClientError.isGrpcTimeout()) {
                 throw predefined.REQUEST_TIMEOUT;
             }
+
+            if (sdkClientError.isInvalidTransactionBody()) {
+                this.logger.error(`${requestIdPrefix} Query object: ${query} Client object: ${client}`);
```

Comment:
This will only end up logging `Query object: [object Object] Client object: [object Object]`.
Further more due to the recursive nature of `client` it would actually fail.
Instead do
```suggestion
                this.logger.error(`${requestIdPrefix} Query object:  ${JSON.stringify(query)}`);
                this.logger.error(`${requestIdPrefix} Client object: ${JSON.stringify({...client, _timer: undefined})}`);
```

I tested this locally with success. Please confirm

Reply:
Me too, works like this.